<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="direc_label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>101</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="x_y_label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>101</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="active_pause_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Parar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="program_label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>101</height>
        </size>
       </property>
       <property name="text">
        <string>Programa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="program_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
